### DISCUSSION TITLE
Uninstall/Remove specific windows appstore application

### DISCUSSION THREAD

**Discussion by Alsen99** [ChrisTitusTech/winutil#1106] [Ideas]
Can you add a way to uninstall specific apps from the store? Like Clipchamp or Cortana. Or is there any reason to keep it?

Well yes there is an option to remove all of the apps but i want to keep camera, voice recoder, xbox app, calculator, calendar, etc. Just easier to remove the apps that I want to remove

**Comment by Hpoonis** [comment]
Has this been addressed?  For the first time today I looked for calculator and it is not there and cnnot be re-installed.
